src/Components/Dashboard/Dashboard.tsx

```diff
@@ -129,6 +129,9 @@ const Dashboard = ({
     };
   
     navigator.geolocation.getCurrentPosition(successCallback, errorCallback);
+
+    setSelectedCampground("")
+    // eslint-disable-next-line
```

Comment:
I see a couple of these eslint disables spread through the code. It's totally fine to keep this - but it can quickly become too much to keep track of. If you find yourself using these on the same types of problems, consider updating your eslint configuration instead. That way, all the changes live in one file instead of spread throughout the codebase.